### ISSUE TITLE
ConvexHull returns duplicated halfspaces

### ISSUE THREAD

**Issue by ferrolho** [scipy/scipy#10442]
### Issue
Calling `ConvexHull` from `scipy.spatial` returns every halfspace _twice_.

### Reproducing code example
```python
#!/usr/bin/python

import numpy as np
from scipy.spatial import ConvexHull

unit_cube = np.array([
    [0.0, 0.0, 0.0],
    [0.0, 1.0, 0.0],
    [1.0, 0.0, 0.0],
    [1.0, 1.0, 0.0],
    [0.0, 0.0, 1.0],
    [0.0, 1.0, 1.0],
    [1.0, 0.0, 1.0],
    [1.0, 1.0, 1.0],
])

hull = ConvexHull(unit_cube)
print(hull.equations)
```

### Output
Notice the duplicated halfspaces:
```
[[-0. -0. -1.  0.]
 [-0. -0. -1.  0.]
 [ 0. -1.  0. -0.]
 [ 0. -1.  0. -0.]
 [ 1. -0. -0. -1.]
 [ 1. -0. -0. -1.]
 [-1. -0. -0.  0.]
 [-1. -0. -0.  0.]
 [ 0.  1. -0. -1.]
 [ 0.  1. -0. -1.]
 [-0. -0.  1. -1.]
 [-0. -0.  1. -1.]]
```

### Scipy/Numpy/Python version information:
Observed in both:
```
('1.2.2', '1.16.4', sys.version_info(major=2, minor=7, micro=12, releaselevel='final', serial=0))
```
```
1.3.0 1.16.4 sys.version_info(major=3, minor=6, micro=9, releaselevel='final', serial=0)
```
> May be occurring in more versions; I did not test on more.

**Comment by ferrolho** [comment]
Related: https://github.com/JuliaPolyhedra/QHull.jl/issues/27
> The fact that the halfspaces are given twice may be due to the fact that each facet is made up of two triangles and QHull returns only simplicial facets

So, this is probably not a bug. Still, might be useful to have some discussion on whether this should be handled differently or not, or at least documented.

**Comment by pv** [comment]
Yes, it reflects the triangulated output, which is currently forced to be on.

It is possible to add an option for controlling whether output is triangulated or not. This should not be very complicated, most of the required things are there; likely `get_hull_facets` would need to be modified to extract also neighbors/equations. In the API, the `simplices` attribute should probably then be set to `None` when the output is not triangulated, and a `facets` attribute added.
